src/commands/Admin/conf.ts

```diff
@@ -1,17 +1,17 @@
 import { SettingsMenu } from '@lib/structures/SettingsMenu';
 import { SkyraCommand, SkyraCommandOptions } from '@lib/structures/SkyraCommand';
 import { PermissionLevels } from '@lib/types/Enums';
+import { LanguageKeys } from '@lib/types/namespaces/LanguageKeys';
 import { codeBlock, toTitleCase } from '@sapphire/utilities';
 import { ApplyOptions, requiredPermissions } from '@skyra/decorators';
 import { configurableSchemaKeys, displayEntry, displayFolder, initConfigurableSchema, isSchemaEntry } from '@utils/SettingsUtils';
 import { KlasaMessage, SettingsFolder } from 'klasa';
 
 @ApplyOptions<SkyraCommandOptions>({
 	aliases: ['settings', 'config', 'configs', 'configuration'],
-	description: (language) => language.get('commandConfServerDescription'),
+	description: (language) => language.get(LanguageKeys.Commands.Admin.ConfServerDescription),
```

Comment:
Since we've removed UserSettings from our klasa fork, we should name this `LanguageKeys.Commands.Admin.ConfDescription`

Reply:
Fixed in 16870cd4.